packages/Search/examples/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -392,13 +391,13 @@ int main( int argc, char *argv[] )
 
         if ( node == "" )
         {
-            typedef KokkosClassic::DefaultNode::DefaultNodeType Node;
+            typedef Kokkos::Serial Node;
```

Comment:
This line is problematic.

Reply:
In what sense?

Reply:
Because Serial is not necessarily enabled.